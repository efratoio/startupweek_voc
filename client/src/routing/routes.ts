export const routeLogin = () => "/login";
export const routeDashboard = () => "/dashboard";
export const routeSignup = () => "/signup";
