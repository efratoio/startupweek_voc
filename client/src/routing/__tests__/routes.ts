import * as routes from "../routes";

["routeLogin", "routeDashboard", "routeSignup"].forEach(route => {
    test(`The route "${route}" returns the correct route`, () => {
        expect(routes[route]()).toMatchSnapshot();
    });
});
