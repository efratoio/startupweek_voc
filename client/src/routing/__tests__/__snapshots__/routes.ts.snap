// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The route "routeDashboard" returns the correct route 1`] = `"/dashboard"`;

exports[`The route "routeLogin" returns the correct route 1`] = `"/login"`;

exports[`The route "routeSignup" returns the correct route 1`] = `"/signup"`;
